libraries/ui-library/src/components/six-select/six-select.tsx

```diff
@@ -448,24 +448,26 @@ export class SixSelect {
 
       this.displayTags = checkedItems.map((item) => {
         return (
-          <six-tag
-            exportparts="base:tag"
-            type="primary"
-            size={this.size}
-            pill={this.pill}
-            clearable
-            onClick={this.handleTagInteraction}
-            onKeyDown={this.handleTagInteraction}
-            onSix-tag-clear={(event) => {
-              event.stopPropagation();
-              if (!this.disabled) {
-                item.checked = false;
-                this.syncValueFromItems();
-              }
-            }}
-          >
-            {this.getItemLabel(item)}
-          </six-tag>
+          <six-tooltip content={item.textContent || ''}>
```

Comment:
Showing the tooltip even if none is needed could be a little cumbersome for the user. For example, the tooltip will always show, when the user wants to remove an item by clicking on the 'x'.

Better would be if the tooltip would only be showed for truncated item, but this might not be simple.

Reply:
Tooltip has been removed completely as we discussed earlier